test/env.js

```diff
@@ -1,10 +1,9 @@
+import { generateIconNames } from '../development/generate-icon-names.js';
 process.env.METAMASK_ENV = 'test';
 
 /**
  * Used for testing components that use the Icon component
  * 'ui/components/component-library/icon/icon.js'
  */
-process.env.ICON_NAMES = {
-  LOADING_FILLED: 'loading-filled',
-  CLOSE_OUTLINE: 'close-outline',
-};
+
+process.env.ICON_NAMES = generateIconNames();
```

Comment:
We can now automatically generate the icon name environment variables using the same `generateIconNames` function yay!